splice_machine/src/main/java/com/splicemachine/derby/stream/function/QuoteTrackingTokenizer.java

```diff
@@ -30,18 +32,84 @@
  * versus columns with values {@code "Foo"}.
  *
  * @author Scott Fines
- *         Date: 9/29/16
+ * Date: 9/29/16
  */
-public class QuoteTrackingTokenizer extends AbstractTokenizer{
+public class QuoteTrackingTokenizer extends AbstractTokenizer {
+
+    private final List<Integer> valueSizeHints;
+    // the line number where a potential multi-line cell starts
+    private int potentialSpaces;
+    private int charIndex;
+    private boolean wasQuoted;
+    // the line number where a potential multi-line cell starts
+    private int quoteScopeStartingLine;
+    private TokenizerState state;
+    private int colIdx;
+    private boolean firstRun;
+    private boolean checkExactSizeFirst = false;
+    private int rowIdx = 0;
+    private List<Integer> exactColumnSizes = null;
+    final static int SCAN_THRESHOLD = 10000;
```

Comment:
This would be better as a configuration parameter, but we can leave it like this

Reply:
Yeah, I totally agree with you, but I don't know how to set the configuration parameters yet :)